cfgov/templates/v1/demo/index.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'v1/demo/layout/base.html' %}
+{% load wagtailcore_tags wagtailimages_tags unescape %}
+
+{% block title %}
+    Consumer Financial Protection Bureau
+{% endblock %}
+
+{% block desc %}
+    Our vision is a consumer finance marketplace that works for American consumers,
+    responsible providers, and the economy as a whole.
+{% endblock %}
+
+{% block content %}
+    <div class="block
+                block__flush-top">
+        {{ self.title }}
+    </div>
+
+    <div class="content-l">
+        <div class="content-l_col-1-2">
+            {% for item in self.molecules %}
+                {{ item|safe }}
+            {% endfor %}
+        </div>
+        <div class="content-l_col-1-2">
+            {% for item in self.organisms %}
+                {{ item|safe }}
```

Comment:
Since the organisms and molecules are being marked safe in their templates, this is unnecessary.